config/routes.rb

```diff
@@ -111,4 +112,3 @@
 #       letters GET    /                                letter_opener_web/letters#index
 #        letter GET    /:id(/:style)(.:format)          letter_opener_web/letters#show
 #               GET    /:id/attachments/:file(.:format) letter_opener_web/letters#attachment
-# rubocop:enable Layout/LineLength
```

Comment:
Was this an accidental removal?  Or does rubocop turn disables on at the end of a file automatically now? 🤔

Reply:
Yep, it was applied automatically for some reason. I'm happy to roll it back, but suspect it will be applied again during upcoming migrations anyway...

Reply:
If you could roll it back, that'd be great!